.travis.yml

```diff
@@ -1,33 +1,18 @@
 language: python
-python:
-  - 2.6
-  - 2.7
+sudo: false
+python: 2.7
+cache:
+  - directories:
+    - ~/.cache/pip
 env:
   matrix:
-  - DJANGO_VERSION=1.3.7
-  - DJANGO_VERSION=1.4.5
-  - DJANGO_VERSION=1.5.1
+    - DJANGO='django~=1.8.0'
   global:
     secure: "ohFDQycYOrveVlvEVwHwws2+xPOvFT2TmyoQk/2fILNpl6tiuI/91XdVv8dQ\n5ZsLXc63LP4CFYbCKkC6jCFrlWePB0aITF35bpGJqXk68JptTcsKF9L5LyHX\ntMqLKAgIBeMs/upSXx6LrKrdwmjmPx0rZL71j+mCN1IeNW/0Fpo="
 install:
-- pip install -q Django==$DJANGO_VERSION --use-mirrors
-- pip install -q Pillow --use-mirrors
-- pip install . --use-mirrors
-- export DEBUG=True
+  - pip install -r requirements.txt --use-wheel
+  - pip install $DJANGO --use-wheel
+  - export DEBUG=True
```

Comment:
Maybe we can try to remove `export DEBUG=True`?
I think we don't need `--use-wheel` as the latest pip does it by default.